DriverService/Data/DriverDAL.cs

```diff
@@ -57,7 +57,19 @@ public void UpdatePosition(int id, Driver obj)
         //Update Accept Order -> Communication with Order Service
         public void UpdateAcceptOrder(int id, Driver obj)
         {
+
             throw new NotImplementedException();
+            // try
+            // {
+            //     var result = _db.Drivers.FirstOrDefault(p => p.Id == id);
+
+            //     if (obj.IsApprove == false)
+            //     Console.WriteLine(" Status driver belum di approve ");
+            // }
+            // catch (System.Exception)
+            // {
+            //     throw;
+            // }
```

Comment:
also, try to avoid committing lines of commented code

Reply:
https://kentcdodds.com/blog/please-dont-commit-commented-out-code

Reply:
oke done mas